scripts/ioc_deploy.py

```diff
@@ -1,35 +1,62 @@
 #!/usr/bin/python3
 """
 ioc-deploy is a script for building and deploying ioc tags from github.
-It will create a shallow clone of your IOC in the standard release area at the correct path and "make" it.
+
+It has two paths: the normal deploy path, and a second path that adjusts
```

Comment:
```suggestion
It has two operation modes: the normal deploy mode, and a second mode that adjusts
```

I think that in a tool that cares so much about the filepath, using "paths" here and elswhere in the docs could be a bit confusing.

Reply:
I'll read the rest of the docs with this in mind and brainstorm some adjustments, but mode is also the wrong word because this is the most technically correct way to describe file permissions (file modes)